package.json

```diff
@@ -14,6 +14,8 @@
         "ios.sauce.simulators.app.eu": "REGION=eu wdio ./config/saucelabs/wdio.ios.simulators.app.conf.ts",
         "ios.sauce.simulators.app.us": "wdio ./config/saucelabs/wdio.ios.simulators.app.conf.ts",
         "ios.browser": "wdio config/wdio.ios.browser.conf.ts",
+        "browser.chrome": "wdio config/wdio.browser.chrome.conf.ts",
```

Comment:
to add firefox, it is basically the same, you'll need a wdio driver + the geckodriver both can be installed with npm